tox.ini

```diff
@@ -247,13 +232,19 @@ commands=
   vagrant up --no-provision {posargs:--provider=virtualbox}
   bash {toxinidir}/tests/scripts/generate_ssh_config.sh {changedir}
 
+  centos7_cluster: ansible-playbook -vv -i {changedir}/hosts {toxinidir}/tests/functional/lvm_setup.yml
```

Comment:
Is there a reason why would wouldn't always run ``lvm_setup.yml`` now?

Reply:
There is not, let me clean that up